src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -20,40 +23,69 @@ export interface ShippingProps {
-          <>
-            <RadioGroup
-              defaultValue="SHIP"
-              onSelect={id => id}
-              options={[
-                { label: "Provide shipping address", id: "SHIP" },
-                { label: "Arrange for pickup", id: "PICKUP" },
-              ]}
-            />
-            <Spacer mb={3} />
-            <Join separator={<Spacer mb={2} />}>
-              <Placeholder height="68px" name="Full Name" />
-              <TwoColumnSplit>
-                <Placeholder height="68px" name="Country" />
-                <Placeholder height="68px" name="Postal Code" />
-              </TwoColumnSplit>
-              <TwoColumnSplit>
-                <Placeholder height="68px" name="Address Line 1" />
-                <Placeholder height="68px" name="Address Line 2 (optional)" />
-              </TwoColumnSplit>
-              <TwoColumnSplit>
-                <Placeholder height="68px" name="City" />
-                <Placeholder height="68px" name="State, province, or region" />
-              </TwoColumnSplit>
-            </Join>
-            <Spacer mb={3} />
-          </>
-        }
-        Sidebar={<Summary mediator={this.props.mediator} />}
-      />
+      <Responsive>
+        {({ xs }) => (
+          <TwoColumnLayout
+            Content={
+              <>
+                <RadioGroup
+                  onSelect={id => id}
+                  options={[
+                    { label: "Provide shipping address", id: "SHIP" },
+                    { label: "Arrange for pickup", id: "PICKUP" },
+                  ]}
+                />
+                <Spacer mb={3} />
+                <Join separator={<Spacer mb={2} />}>
+                  <Placeholder height="68px" name="Full Name" />
+                  <TwoColumnSplit>
+                    <Placeholder height="68px" name="Country" />
+                    <Placeholder height="68px" name="Postal Code" />
+                  </TwoColumnSplit>
+                  <TwoColumnSplit>
+                    <Placeholder height="68px" name="Address Line 1" />
+                    <Placeholder
+                      height="68px"
+                      name="Address Line 2 (optional)"
+                    />
+                  </TwoColumnSplit>
+                  <TwoColumnSplit>
+                    <Placeholder height="68px" name="City" />
+                    <Placeholder
+                      height="68px"
+                      name="State, province, or region"
+                    />
+                  </TwoColumnSplit>
+                  {!xs && (
+                    <Link to={`/order2/${order.id}/payment`}>
+                      <Button size="large" width="100%">
+                        Continue
+                      </Button>
+                    </Link>
+                  )}
+                </Join>
+                <Spacer mb={3} />
+              </>
+            }
+            Sidebar={
+              <Summary mediator={this.props.mediator}>
```

Comment:
Since the `mediator` is stored in the global app state, I was planning to just `Subscribe` from the summary component (so you wouldn't have to pass it here). But not a blocker for this, I can update in a subsequent PR.